jumanji/environments/combinatorial/binpack/env.py

```diff
@@ -103,20 +105,21 @@ class BinPack(Environment[State]):
 
     def __init__(
         self,
-        instance_generator: InstanceGenerator = None,
+        instance_generator: str = "toy",
         obs_num_ems: int = 60,
         reward_fn: RewardFn = sparse_linear_reward,
```

Comment:
Outside of the scope of this PR, we could use a similar factory for the reward function to avoid passing complex objects to the constructor. Having `int`, `float` and `str` as constructor arguments facilitate a lot.

Reply:
I'll let you create an (internal?) issue and mark this as resolved.